File: app/src/main/java/com/qfq/tainzhi/videoplayer/my_media/DataSource.java
package com.qfq.tainzhi.videoplayer.my_media;

import android.net.Uri;

/**
 * @author: tainzhi
 * @mail: qfq61@qq.com
 * @date: 2019-11-10 19:34
 * @description:
 **/

public class DataSource {
	private String url;
	private String title;
	private Uri uri;
	
	public String getUrl() {
		return url;
	}
	
	public void setUrl(String url) {
		this.url = url;
	}
	
	public DataSource(String url, String title) {
		this.url = url;
		this.title = title;
	}
	
	public DataSource(Uri uri) {
		this.uri = uri;
	}
	
	public Uri getUri() {
		return uri;
	}
	
	public void setUri(Uri uri) {
		this.uri = uri;
	}
	
	public String getTitle() {
		return title;
	}
	
	public void setTitle(String title) {
		this.title = title;
	}
}

File: QMediaPlayer/src/main/java/com/tanzhi/qmediaplayer/DataSource.kt
package com.tanzhi.qmediaplayer

import android.net.Uri

/**
 * @author: tainzhi
 * @mail: qfq61@qq.com
 * @date: 2019-11-10 19:34
 * @description:
 */
class DataSource {
    var url: String? = null
    var title: String? = null
    var uri: Uri? = null

    constructor(url: String?, title: String?) {
        this.url = url
        this.title = title
    }

    constructor(uri: Uri?) {
        this.uri = uri
    }

}